content/en/docs/reference/glossary/resource-claim-parameters.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Resource Claim Parameters
```

Comment:
Following our style guide, this would be “ResourceClaimParameters” (no spaces). Same for other API kinds.

Reply:
I agree about spelling this in text somewhere, but looking at the rest of glossary items, they seem to be spelled with space in title, and with hyphen in id, I just tried to keep it consistent.

Reply:
- No spaces in titles
- ID can be either kebab-case or (preferred) matching the API kind

See https://github.com/kubernetes/website/pull/43742#discussion_r1486565400

Reply:
Updated glossary items.